<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>215</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>215</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnPlay">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrevious">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QLabel" name="music_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="artist_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QSlider" name="volume">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>20</width>
      <height>91</height>
     </rect>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TickPosition::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMenu">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMute">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbout">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentTime">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="remainTime">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionAbrir_Diret_rio">
   <property name="text">
    <string>Abrir Diretório</string>
   </property>
  </action>
  <action name="actionAbrir_Arquivo">
   <property name="text">
    <string>Abrir Arquivo</string>
   </property>
  </action>
  <action name="actionMostrar_udio">
   <property name="text">
    <string>Mostrar Áudio</string>
   </property>
  </action>
  <action name="actionMostrar_VU">
   <property name="text">
    <string>Mostrar VU</string>
   </property>
  </action>
  <action name="actionMostrar_Informa_es">
   <property name="text">
    <string>Mostrar Informações</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
